From 0337a2dd4a7f8394189f816df94e0f3219aaeeca Mon Sep 17 00:00:00 2001
From: Tom Pollard <tom.pollard@codethink.co.uk>
Date: Fri, 12 Jun 2026 12:55:00 +0100
Subject: [PATCH 08/10] Update elements/include/flatpak.yml to 1.18.0-0

Updates elements/include/flatpak.yml to 1.18.0-0 (5de2b58)
Previous version was 1.16.6-0 (e761a88)

Configrable `http_backend` build option dropped as support for
libsoup removed & only libcurl is supported as of 1.17.1:

https://github.com/flatpak/flatpak/commit/e4edecdef5b0aacad5af3be5
07dcff37e850162e
---
 elements/components/flatpak-minimal.bst | 1 -
 elements/components/flatpak.bst         | 1 -
 elements/include/flatpak.yml            | 2 +-
 3 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/elements/components/flatpak-minimal.bst b/elements/components/flatpak-minimal.bst
index 536b8627c..ea074072e 100644
--- a/elements/components/flatpak-minimal.bst
+++ b/elements/components/flatpak-minimal.bst
@@ -29,7 +29,6 @@ variables:
     -Dsystem_helper=disabled
     -Dxauth=disabled
     -Dsystem_bubblewrap=bwrap
-    -Dhttp_backend=curl
     -Ddconf=disabled
     -Dmalcontent=disabled
     -Dselinux_module=disabled
diff --git a/elements/components/flatpak.bst b/elements/components/flatpak.bst
index 9989a2a9e..de592d5b8 100644
--- a/elements/components/flatpak.bst
+++ b/elements/components/flatpak.bst
@@ -35,7 +35,6 @@ variables:
   meson-local: >-
     -Dgtkdoc=disabled
     -Dsystem_bubblewrap=bwrap
-    -Dhttp_backend=curl
     -Ddconf=disabled
     -Dmalcontent=disabled
     -Dselinux_module=disabled
diff --git a/elements/include/flatpak.yml b/elements/include/flatpak.yml
index b2620c34b..d854786e9 100644
--- a/elements/include/flatpak.yml
+++ b/elements/include/flatpak.yml
@@ -2,4 +2,4 @@ sources:
 - kind: git_repo
   url: github:flatpak/flatpak.git
   track: 1.1[02468]*
-  ref: 1.16.6-0-ge761a8885453c217a931281092a641ebbdd0a0c6
+  ref: 1.18.0-0-g5de2b58bdafde88515d9d6b9189bd1cfd3b88c46
-- 
2.54.0

